resources/views/layouts/header.blade.php

```diff
@@ -68,30 +67,69 @@
                            </p>
                         </div>
                      </div>
-                  </div> 
-            </div>
+                  </div>
+               </div>
+               <!-- Active burger menu-->
+               <div class="navbar-menu" id="Options">
```

Comment:
Haha I'm being very picky, please change to lowercase for id Options - in the name of consistency